Tests/LogtoTests/LogtoUtilitiesTests.swift

```diff
@@ -0,0 +1,17 @@
+@testable import Logto
+import XCTest
+
+final class LogtoUtilitiesTests: XCTestCase {
+    func fromBase64(string: String) -> String? {
+        guard let data = Data(base64Encoded: string) else {
+            return nil
+        }
+
+        return String(data: data, encoding: .utf8)
```

Comment:
just wondering what's the default encoding method for string in swift? do we have to specify it everytime forming a string?

Reply:
![image](https://user-images.githubusercontent.com/14722250/148520849-c44ff962-f89e-4078-841b-cecee9c8c6a8.png)

unfortunately there's no default encoding when transforming `Data` to `String` in swift